perf/ListPool.Benchmarks/ListPoolContainsBenchmark.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections.Generic;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Order;
+
+namespace ListPool.Benchmarks
+{
+    [RankColumn]
+    [Orderer(SummaryOrderPolicy.FastestToSlowest)]
+    [MemoryDiagnoser]
+    [GcServer(true)]
+    [GcConcurrent]
+    public class ListPoolContainsBenchmark
+    {
+        [Params(10, 100, 1000, 10000)]
+        public int N { get; set; }
+
+        private List<int> _list;
+        private ListPool<int> _listPool;
+
+        [IterationSetup]
+        public void IterationSetup()
+        {
+            _list = new List<int>(N);
+            _listPool = new ListPool<int>(N);
+
+            for (int i = 1; i <= N; i++) 
+            {
+                _list.Add(i);
+                _listPool.Add(i);
+            }
+        }
+
+        [IterationCleanup]
+        public void IterationCleanup()
+        {
+            _listPool.Dispose();
+        }
+
+        [Benchmark(Baseline = true)]
+        public void List()
+        {
+            _list.Contains(1);
+            _list.Contains(N / 2);
```

Comment:
I think just picking one should give us good stats.

Reply:
done :)